codec-http2/src/main/java/io/netty/handler/codec/http2/WeightedFairQueueByteDistributor.java

```diff
@@ -390,17 +732,19 @@ public String toString() {
         }
 
         private void toString(StringBuilder sb) {
-            sb.append("{stream ").append(stream.id())
+            sb.append("{streamId ").append(streamId)
                     .append(" streamableBytes ").append(streamableBytes)
                     .append(" activeCountForTree ").append(activeCountForTree)
-                    .append(" priorityQueueIndex ").append(priorityQueueIndex)
+                    .append(" pseudoTimeQueueIndex ").append(pseudoTimeQueueIndex)
                     .append(" pseudoTimeToWrite ").append(pseudoTimeToWrite)
                     .append(" pseudoTime ").append(pseudoTime)
                     .append(" flags ").append(flags)
-                    .append(" queue.size() ").append(queue.size()).append("} [");
+                    .append(" pseudoTimeQueue.size() ").append(pseudoTimeQueue.size()).append("} [")
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) StringBuffer (or StringBuilder).append is called 2 consecutive times with literal Strings. Use a single append with a single combined String. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=pmd%3AConsecutiveLiteralAppends)